sources/matomo/README.md

```diff
@@ -0,0 +1,74 @@
+# Matomo
+
+Matomo is a free and open-source web analytics platform that allows website owners and businesses to gain detailed insights into their website and application performance. With this verified source, you can easily extract and seamlessly load data from Matomo to your preferred destination. This verifies source supports the following endpoints:
+
+| Endpoint | Description |
+| --- | --- |
+| visits | the visits in the given site ID for the given period |
```

Comment:
these are not endpoints, but sources

it is not easy to understand what's happening in this project, so i suggest to keep it like that, but rename  `This verifies source supports the following endpoints:` to `This verifies source supports the following sources:`

Reply:
or may be it makes sense to remove it altogether

Reply:
yeah maybe just describe what resources be found, afaik
event data:
- visit events
- users of those visit events, with info about their session
aggregated data:
- reports

Reply:
please check 👍

Reply:
the intention here was to separate
- daily running (aggregated reports on date)
from 
- any-time top up with events
- but users require merge so maybe those not that often for cost reason